ovh/provider.go

```diff
@@ -46,6 +46,8 @@ func Provider() terraform.ResourceProvider {
 			// New naming schema (issue #23)
 			"ovh_cloud_region":  dataSourcePublicCloudRegion(),
 			"ovh_cloud_regions": dataSourcePublicCloudRegions(),
+			"ovh_credit_card":   dataSourceCreditCard(),
```

Comment:
could we map the ovh api by prefixing datasources by "ovh_me_" ?
pls ? and rename the go functions accordingly ?

thanks a lot